cypress/e2e/michael.cy.ts

```diff
@@ -0,0 +1,16 @@
+describe('drag', () => {
+    xit('click twice', () => {
```

Comment:
Test cases submitted are taken as prod code. All tests should be enabled by default.